internal/gengapic/gengapic.go

```diff
@@ -153,6 +138,37 @@ func (g *generator) gen(serv *descriptor.ServiceDescriptorProto) error {
 		}
 	}
 
+	return nil
+}
+
+// gen generates client for the given service.
+func (g *generator) gen(serv *descriptor.ServiceDescriptorProto) error {
+	servName := pbinfo.ReduceServName(*serv.Name, g.opts.pkgName)
+
+	g.clientHook(servName)
+	if err := g.clientOptions(serv, servName); err != nil {
+		return err
+	}
+	if err := g.makeClients(serv, servName); err != nil {
+		return err
+	}
+
+	for _, v := range g.opts.transports {
+		switch v {
+		case grpc:
+			if err := g.genGrpcMethods(serv, servName); err != nil {
+				return err
+			}
+		case rest:
+			if err := g.genRestMethods(serv, servName); err != nil {
```

Comment:
nit: capitalize `genRESTMethods`

similarly with `genGrpcMethods` above, though consistency with the rest of the file/repo overrides.

Reply:
Done